src/test/kotlin/no/nav/syfo/brev/arbeidstaker/ArbeidstakerBrevApiSpek.kt

```diff
@@ -302,6 +305,160 @@ class ArbeidstakerBrevApiSpek : Spek({
                         }
                     }
                 }
+                describe("Happy path for arbeidstaker som har byttet fnr") {
+                    val validTokenSelvbetjeningOldFnr = generateJWTTokenx(
+                        audience = externalMockEnvironment.environment.tokenxClientId,
+                        issuer = externalMockEnvironment.wellKnownSelvbetjening.issuer,
+                        pid = ARBEIDSTAKER_TREDJE_FNR.value,
+                    )
+                    val newDialogmoteDTO = generateNewDialogmoteDTO(ARBEIDSTAKER_FJERDE_FNR)
+                    val urlMote = "$dialogmoteApiV2Basepath/$dialogmoteApiPersonIdentUrlPath"
+                    val urlArbeidstakerMoterList = arbeidstakerBrevApiPath
+
+                    it("should return OK if request is successful") {
```

Comment:
Dette var en kjempestor test, er den mulig å dele opp i flere it-er?

Reply:
Antakelig mulig, men i så fall så burde det samme gjøres med det andre testene her. Ikke helt opplagt for meg om det vil være bedre eller ikke, så jeg synes vi skal tenke på det - og evnt refaktorere denne testklassen i en egen PR.